testing/bsg_dmc/bsg_tag_boot_rom.tr

```diff
@@ -0,0 +1,309 @@
+0001___1___100000_1_0001___00000000
+
+#SEND  en   id=33   d l=3    {ds_val, reset}
+0001___1___100001_1_0011___00000001
+0001___1___100001_1_0011___00000000
+
+#SEND  en   id=34   d l=2    {sel}
+0001___1___100010_1_0010___00000000
+
+### Set delay line triggers low
+
+#SEND  en   id=5  d l=1    {trigger}
+0001___1___000101_1_0001___00000000
+#SEND  en   id=6  d l=1    {trigger}
+0001___1___000110_1_0001___00000000
+#SEND  en   id=7  d l=1    {trigger}
+0001___1___000111_1_0001___00000000
+#SEND  en   id=8  d l=1    {trigger}
+0001___1___001000_1_0001___00000000
+
+### Program the delay lines
+
+#SEND  en   id=1  d l=5    {adt, cdt, fdt}
+0001___1___000001_1_0101___00011011
+#SEND  en   id=2  d l=5    {adt, cdt, fdt}
+0001___1___000010_1_0101___00011011
+#SEND  en   id=3  d l=5    {adt, cdt, fdt}
+0001___1___000011_1_0101___00011011
+#SEND  en   id=4  d l=5    {adt, cdt, fdt}
+0001___1___000100_1_0101___00011011
+
+### Trigger delay lines
+
+#SEND  en   id=5  d l=1    {trigger}
+0001___1___000101_1_0001___00000001
+0001___1___000101_1_0001___00000000
+#SEND  en   id=6  d l=1    {trigger}
+0001___1___000110_1_0001___00000001
+0001___1___000110_1_0001___00000000
+#SEND  en   id=7  d l=1    {trigger}
+0001___1___000111_1_0001___00000001
+0001___1___000111_1_0001___00000000
+#SEND  en   id=8  d l=1    {trigger}
+0001___1___001000_1_0001___00000001
+0001___1___001000_1_0001___00000000
+
+### Async delay line reset to get things moving
+
+#SEND  en   id=0  d l=1    {async_reset}
+0001___1___000000_1_0001___00000000
+0001___1___000000_1_0001___00000001
+0001___1___000000_1_0001___00000000
+
+### Set downsamples and reset
+#
+#SEND  en   id=9  d l=3    {ds_val, reset}
+0001___1___001001_1_0011___00000001
+0001___1___001001_1_0011___00000000
+
+# Reset DMC configs and set dmc_p
+#SEND  en   id=10  d l=8    {trefi[7:0]}
+0001___1___001010_1_1000___11111111
+#SEND  en   id=11  d l=8    {trefi[15:8]}
+0001___1___001011_1_1000___00000011
+#SEND  en   id=12  d l=8    {trfc, tmrd}
+0001___1___001100_1_1000___11110001
+#SEND  en   id=13  d l=8    {trp, trc}
+0001___1___001101_1_1000___00111011
+#SEND  en   id=14  d l=8    {trrd, tras}
+0001___1___001110_1_1000___00101000
+#SEND  en   id=15  d l=8    {twr, trcd}
+0001___1___001111_1_1000___10110010
+#SEND  en   id=16  d l=8    {trtp, twtr}
+0001___1___010000_1_1000___10101001
+#SEND  en   id=17  d l=7    {dqs_sel_cal, cas}
+0001___1___010001_1_0111___00110011
+#SEND  en   id=18  d l=8    {row_width, col_width}
+0001___1___010010_1_1000___11101011
+#SEND  en   id=19  d l=8    {bank_pos, bank_width}
+0001___1___010011_1_1000___01100110
+#SEND  en   id=20  d l=8    {init_cycles[7:0]}
+0001___1___010100_1_1000___01001010
+#SEND  en   id=21  d l=8    {init_cycles[15:8]}
+0001___1___010101_1_1000___10111100
+
+#SEND  en   id=22  d l=8    {rd_calib_cycles[7:0]}
+0001___1___010110_1_1000___11111111
+#SEND  en   id=23  d l=8    {rd_calib_cycles[15:8]}
+0001___1___010111_1_1000___00000010
```

Comment:
Maybe add a comment saying: rd_calib_cycles = 0x02ff setup only works for UI clock frequency above 2 MHz, lower frequencies will need more read calibration cycles